requirements.txt

```diff
@@ -13,10 +13,18 @@ pytest
 pytest-cov
 raven
 requests
+requirements-parser
 scipy
 scikit-image
 scikit-learn
 tensorflow==1.5.0
+# PyTorch's Linux/Windows distribution is very large due to its GPU support,
+# but we only need that for training models. For users, use the CPU-only version
+# (only available directly from the PyTorch project).
+# The OS X version has never had GPU support, so doesn't need the workaround.
+-f https://download.pytorch.org/whl/cpu/torch_stable.html
+torch==1.5.0+cpu; sys_platform != "darwin"
+torch==1.5.0; sys_platform == "darwin"
```

Comment:
You should add torchvision==0.6.0(+cpu for linux) which is needed in all cases.

Reply:
Thank you for catching this. I've patched it and it's running CI now.